src/components/SpiceList.js

```diff
@@ -7,6 +8,14 @@ class SpiceList extends React.Component {
   renderSpices() {
     // TODO: update this function to return an array of SpiceItem components
     // If you get an error message about a 'key' prop, make sure to fix it before submitting your code!
+    const allSpices = this.props.spices;
+
+    return allSpices.map(spice => {
+      const {id, title, image, description, notes} = spice;
+
+      return <SpiceItem key={id} title={title} image={image} description={description} notes={notes}/>
+    })
+
```

Comment:
Nice use of destructuring! It's also good that you passed down each key from the spice as a separate prop instead of just passing down the whole object, that means you have a better interface to work with for the SpiceItem's props. You can also use a shorthand syntax with the spread operator to pass down all keys from an object as props:

```jsx
this.props.spices.map(spice => <SpiceItem key={spice.id} {...spice} />)
```

I'd say to be careful with that spread operator because it gives you less control over what props you're passing down, but it can be useful in some cases. In your case, since you don't need to pass down the ID to the SpiceItem component, your approach is better.